quickjs/src/native/Context.cpp

```diff
@@ -163,6 +165,42 @@ jbyteArray Context::compile(JNIEnv* env, jstring source, jstring file) {
   return result;
 }
 
+jobject Context::memoryUsage(JNIEnv* env) {
+  JSMemoryUsage jsMemoryUsage;
```

Comment:
I'm assuming this is stack allocated and doesn't need any kind of explicit free-ing.

Reply:
You got it.